pkg/virt-launcher/virtwrap/network/cloud-init-network.go

```diff
@@ -0,0 +1,284 @@
+// pkg/virt-launcher/virtwrap/network/cloud-init-network.go currently adds
+// support to configure SR-IOV interfaces within a VM through cloud-init
+// network version 1 configuration. Other interface types such as bridge
```

Comment:
It will just require a lot more work. The aim of this initial request is to support SR-IOV that was recently added. I don't want perfect being the enemy of good in getting this request merged.